src/Core/src/Platform/Android/TransformationExtensions.cs

```diff
@@ -28,12 +28,22 @@ public static void UpdateScale(this AView platformView, IView view)
 
 		public static void UpdateScaleX(this AView platformView, IView view)
 		{
-			platformView.ScaleX = (float)(view.Scale * view.ScaleX);
+			var scale = view.Scale;
+
+			if (double.IsNaN(scale))
```

Comment:
Fix crash animating  several times in a rowtimes Frame scale
https://github.com/xamarin/Xamarin.Forms/pull/14606